Cue/app/tabs/library/CardListView.js

```diff
@@ -10,50 +11,107 @@ import EmptyView from '../../common/EmptyView'
 import CardListViewRow from './CardListViewRow'
 import ListViewHairlineSeparator from '../../common/ListViewHairlineSeparator'
 
+const styles = {
+  liftedRowStyle: {
+    opacity: 1,
+    transform: [{scale: 1.02}],
+    backgroundColor: 'white',
+    overflow: 'visible',
```

Comment:
Yeah, it's to stop the shadows (which extend outside the bounds of the view) from being clipped.